app/src/main/java/com/palone/planahead/screens/taskEdit/ui/taskTypeSelection/typeProperties/choreProperties/ChoreProperties.kt

```diff
@@ -22,43 +22,44 @@ fun ChoreProperties(
     dateAndTime: LocalDateTime,
     selectedIntervalValue: Int,
     selectedIntervalUnit: ChronoUnit,
-    onChooseDateAndTimeEvent: () -> Unit,
+    onNewDateTime: (dateTime: LocalDateTime) -> Unit,
```

Comment:
In the context of chore, this `dateTime` has a business meaning - I would find a name for it (e.g. deadlineDateTime, startDateTime) and use it here.